benchmarks/zlib_zlib_uncompress_fuzzer/checksum_fuzzer.c

```diff
@@ -0,0 +1,67 @@
+#include <stdio.h>
```

Comment:
What should we do about licenses? Everything in OSS-Fuzz should have a license header but some don't. For now I added benchmarks/ to the list of ignored directories for the license check.

Reply:
Why not add the licence? They were google code in the oss-fuzz repo with the same licence.